### ISSUE TITLE
Training loss and measure metrics

### ISSUE THREAD

**Issue by wangjiawen2013** [tracel-ai/burn#3171]
Hi,
I have a silly question. We usually use loss (such as crossentropyloss/mseloss) for backpropagation during training, while use recall, precsion, F1-score as performance metrics. Why not try the other way round ? I mean, use recall, precsion, F1-score for backpropagation during training and use loss as performance metrics.
